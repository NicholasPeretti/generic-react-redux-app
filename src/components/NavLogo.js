import React from 'react'
import logo from '../logo.png'
import Style from '../style'
import styled from 'styled-components'

const StyledContainer = styled.div`
  display: flex;
  justify-content: center;
`
const StyledImage = styled.img`
  max-height: 30px;
  margin: ${Style.spacing.padding3} 0px;
`

const NavLogo = () => (
  <StyledContainer>
    <StyledImage src={logo} alt="Auto1" />
  </StyledContainer>
)

export default NavLogo
