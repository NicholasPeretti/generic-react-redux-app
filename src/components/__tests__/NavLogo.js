import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import NavLogo from '../NavLogo'

describe('NavLogo', () => {
  PresentationalTest(NavLogo)

  test('Should match snapshot', () => {
    const wrapper = shallow(<NavLogo/>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
