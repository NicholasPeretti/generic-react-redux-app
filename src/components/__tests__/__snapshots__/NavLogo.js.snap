// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLogo Should match snapshot 1`] = `"<div class=\\"sc-bdVaJa NFzwl\\"><img src=\\"logo.png\\" alt=\\"Auto1\\" class=\\"sc-bwzfXH foBZsz\\"/></div>"`;
